public/style.css

```diff
@@ -684,7 +684,8 @@ div.thumb:hover svg {
 
 .inboxCard {
   /* border: 1px solid black; */
-  height: 50px;
+  max-height: 100px;
+  height: 100%;
```

Comment:
this would just create a thin wrapper around the text instead of looking like a card you'd click on.  no need to change though, we'll need to go back and style this anyway